DockerfileMakePackage

```diff
@@ -45,6 +45,7 @@ RUN if [ "$ov_use_binary" == "1" ] ; then true ; else exit 0 ; fi ; find /opt/in
 RUN if [ "$ov_use_binary" == "1" ] ; then true ; else exit 0 ; fi ; find /opt/intel/openvino/deployment_tools/inference_engine/external/ -iname '*.so*' -exec cp -v {} /ovms_release/lib/ \;
 RUN if [ "$ov_use_binary" == "1" ] ; then true ; else exit 0 ; fi ; find /opt/intel/openvino/deployment_tools/ngraph/lib/ -iname '*.so*' -exec cp -v {} /ovms_release/lib/ \;
 RUN if [ "$ov_use_binary" == "1" ] ; then true ; else exit 0 ; fi ; find /opt/intel/openvino/deployment_tools/inference_engine/external/ -iname '*.so*' -exec cp -v {} /ovms_release/lib/ \;
+RUN find /usr/lib64/ -iname 'libcrypto.so.*' -exec cp -v {} /ovms_release/lib/ \;
```

Comment:
Is using the dot at the end of name `libcrypto.so.*` intentional? Most commonly there will be also a library with name `libcrypto.so`.

Example from my dev machine:
```bash
$ find /usr/lib64/ -iname 'libcrypto.so.*'
/usr/lib64/libcrypto.so.1.0.2o
/usr/lib64/libcrypto.so.10
/usr/lib64/libcrypto.so.1.1.1g
/usr/lib64/libcrypto.so.1.1

$ find /usr/lib64/ -iname 'libcrypto.so*'
/usr/lib64/libcrypto.so.1.0.2o
/usr/lib64/libcrypto.so.10
/usr/lib64/libcrypto.so.1.1.1g
/usr/lib64/libcrypto.so.1.1
/usr/lib64/libcrypto.so
```

Also cp will copy file content when occurs a symbolic link:
```bash
$ TMP_DIR=$(mktemp -d)

$ find /usr/lib64/ -iname 'libcrypto.so.*' -exec cp -v {} $TMP_DIR \;
'/usr/lib64/libcrypto.so.1.0.2o' -> '/tmp/tmp.jP5Py3LA9B/libcrypto.so.1.0.2o'
'/usr/lib64/libcrypto.so.10' -> '/tmp/tmp.jP5Py3LA9B/libcrypto.so.10'
'/usr/lib64/libcrypto.so.1.1.1g' -> '/tmp/tmp.jP5Py3LA9B/libcrypto.so.1.1.1g'
'/usr/lib64/libcrypto.so.1.1' -> '/tmp/tmp.jP5Py3LA9B/libcrypto.so.1.1'

$ ll $TMP_DIR
total 19M
drwxrwxr-x  2 kromanow kromanow  120 Dec 17 15:33 .
drwxrwxrwt 66 root     root     2.7K Dec 17 15:33 ..
-rwxr-xr-x  1 kromanow kromanow 5.1M Dec 17 15:33 libcrypto.so.10
-rwxr-xr-x  1 kromanow kromanow 5.1M Dec 17 15:33 libcrypto.so.1.0.2o
-rwxr-xr-x  1 kromanow kromanow 4.2M Dec 17 15:33 libcrypto.so.1.1
-rwxr-xr-x  1 kromanow kromanow 4.2M Dec 17 15:33 libcrypto.so.1.1.1g
```

To fix all above you could perform:
```bash
find /usr/lib64/ -iname 'libcrypto.so*' -exec cp -vP {} /ovms_release/lib/ \;
```

Example:
```bash
$ find /usr/lib64/ -iname 'libcrypto.so*' -exec cp -vP {} $TMP_DIR \;
'/usr/lib64/libcrypto.so.1.0.2o' -> '/tmp/tmp.jP5Py3LA9B/libcrypto.so.1.0.2o'
'/usr/lib64/libcrypto.so.10' -> '/tmp/tmp.jP5Py3LA9B/libcrypto.so.10'
'/usr/lib64/libcrypto.so.1.1.1g' -> '/tmp/tmp.jP5Py3LA9B/libcrypto.so.1.1.1g'
'/usr/lib64/libcrypto.so.1.1' -> '/tmp/tmp.jP5Py3LA9B/libcrypto.so.1.1'
'/usr/lib64/libcrypto.so' -> '/tmp/tmp.jP5Py3LA9B/libcrypto.so'

$ ll $TMP_DIR
total 9.2M
drwxrwxr-x  2 kromanow kromanow  140 Dec 17 15:37 .
drwxrwxrwt 66 root     root     2.7K Dec 17 15:35 ..
lrwxrwxrwx  1 kromanow kromanow   19 Dec 17 15:37 libcrypto.so -> libcrypto.so.1.1.1g
lrwxrwxrwx  1 kromanow kromanow   19 Dec 17 15:37 libcrypto.so.10 -> libcrypto.so.1.0.2o
-rwxr-xr-x  1 kromanow kromanow 5.1M Dec 17 15:37 libcrypto.so.1.0.2o
lrwxrwxrwx  1 kromanow kromanow   19 Dec 17 15:37 libcrypto.so.1.1 -> libcrypto.so.1.1.1g
-rwxr-xr-x  1 kromanow kromanow 4.2M Dec 17 15:37 libcrypto.so.1.1.1g
```